SpotMenu/AppDelegate.swift

```diff
@@ -95,10 +108,15 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         updateHidden()
     }
     
-    
-    
-    
+
     // MARK: - Popover
+    func openSite(_ sender: NSMenuItem) {
+//        NSApplicationDelegate.open(url, options: [:], completionHandler: nil)
```

Comment:
i fixed this issue in commit e86e0f042a44ac3c919835cf9ba1729a6d5a386c , do you think it's ok?